cloud/alias.js

```diff
@@ -6,23 +6,22 @@ module.exports.generateName = generateName;
 module.exports.getActive = getActive;
 module.exports.deactivate = deactivate;
 
-var adjectives = require('../cloud/adjectives.js');
-var animals = require('../cloud/animals.js');
+const adjectives = require('../cloud/adjectives.js'),
+    animals = require('../cloud/animals.js'),
+    query = new Parse.Query("Alias");
```

Comment:
what happens if i have
```
function f() {
  query.equalTo("active", true);
}

function g() {
  query.equalTo("userId", "blahblhblah");
  return query.find();
}
```

won't `g()` return something different depending on whether or not I call `f()`